diagnostic.md

```diff
@@ -7,40 +7,45 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:sstone72389/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+ensure you are in the correct directory and create a new branch
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
-
 ```sh
-<!-- Remove this comment and place your answer here. -->
+Either: git branch response, git checkout response
+or all in one: git checkout -b response
 ```
-
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+I am not 100% as to the context of this question. To see the changes on the commit,
+you would use git log. To use the cmd line to view actual changes you can
+concatenate.
```

Comment:
`git log` would show a history of commits. What we're looking for here is `git status`, which will just show us which files and directories have been changed